pkg/handlers/subscription_handler.go

```diff
@@ -308,6 +310,11 @@ func (w *webhookResponseWrapper) WriteHeader(status int) {
 
 // New returns new handler
 func New(cfg Config) *Handler {
+	var requestTimeout int64
+	if requestTimeout = cfg.RouterConfig.RequestTimeout; cfg.RouterConfig.RequestTimeout <= 0 {
+		requestTimeout =
+			int64(^uint32(0) >> 1)
+	}
```

Comment:
```suggestion
	h := Handler{
		Schema:   cfg.Schema,
		graphiql: cfg.GraphiQL,
		pretty:   cfg.Pretty,
		upgrader: websocket.Upgrader{
			ReadBufferSize:    1024,
			WriteBufferSize:   1024,
			EnableCompression: true,
		},
		rootObjectFn: cfg.RootObjectFn,
		rootObjectFn:   cfg.RootObjectFn,
	}
	switch requestTimeout: = cfg.RouterConfig.RequestTimeout; {
	    case nil:
	             h.requestTimeout = time.Second * 30
	    case *requestTimeout <= 0:
	             h.requestTimeout = time.Hour * 24
	    default:
	             h.requestTimeout = *requestTimeout
	}
```

We need to keep the default, if not specified timeout equal to time.Second * 30 for backwards compatibility, and while I personally do not begrudge a bit of bit hacking from time time when necessary, this place is definitely not one of those necessary ones.